dataset/src/main/kotlin/org/jetbrains/kotlinx/dl/dataset/preprocessor/image/ImagePreprocessing.kt

```diff
@@ -21,37 +20,16 @@ public class ImagePreprocessing {
     /** */
     public lateinit var load: Loading
 
-    /** */
-    public lateinit var crop: Cropping
-
-    /** */
-    public lateinit var resize: Resize
-
-    /** */
-    public lateinit var rotate: Rotate
-
-    /** */
-    public lateinit var save: Save
-
-    /** True, if [crop] is initialized. */
-    public val isCropInitialized: Boolean
-        get() = ::crop.isInitialized
-
-    /** True, if [resize] is initialized. */
-    public val isResizeInitialized: Boolean
-        get() = ::resize.isInitialized
-
-    /** True, if [rotate] is initialized. */
-    public val isRotateInitialized: Boolean
-        get() = ::rotate.isInitialized
-
     /** True, if [load] is initialized. */
     public val isLoadInitialized: Boolean
         get() = ::load.isInitialized
 
-    /** True, if [save] is initialized. */
-    public val isSaveInitialized: Boolean
-        get() = ::save.isInitialized
+    /** */
+    internal val operations = mutableListOf<ImagePreprocessor>()
+
+    public fun addOperation(operation: ImagePreprocessor) {
+        operations.add(operation)
```

Comment:
Yeah, thanks, I'd really like to see the removal of these functions, no needs now